neqo-http3/src/transaction_server.rs

```diff
@@ -4,6 +4,9 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// because of receive() function.
+#![allow(clippy::too_many_lines)]
```

Comment:
Can we put this on the function directly?  And open a bug to determine whether fixing it is worthwhile?